docs/Medical/Advanced_Medical.md

```diff
@@ -0,0 +1,57 @@
+# Advanced ACE3 Medical
+ This guide is aimed primarily towards players who have used ACE medical previously and would like to become a good medic for the unit.
+
+## The Medics Mindset
+ As a medic, your main responsibility is to keep your team alive and fighting. In intense combat situations, you need to balance two things: locating and reviving your fallen comrades, and avoiding enemy fire. If you get hit, your team will lose its only medic. Sometimes, you have to make hard choices and let someone go instead of risking your own life and the team's survival. Try to stay calm and professional at all times.
+
+### Why play as a medic
+ A medic is not only a skilled shooter, but also a lifesaver who can heal and revive their teammates in the heat of battle. The medic also has to communicate with other players and coordinate their actions to ensure the safety and survival of their squad. Playing as a medic in Arma is a challenging and satisfying experience that requires teamwork, skill, and dedication.
+
+### What makes a good medic
+ A good medic is someone who can communicate, learn, and enjoy. Communication with your squad allows you to coordinate your actions, avoid risks, and provide timely and effective medical support. Learning the ACE3 medical system is essential. You need to diagnose, treat, and stabilize various injuries, and use the medical equipment and supplies properly. Enjoyment of the role of a medic is also important, as it gives you fun and challenges, and improves your skills and teamwork.
+
+### Bedside manner
+ 
+
+## Working with a team
+### Communication
+### Chain of Command
+### Placement and Movement
+### Working with other medics
+## Dealing with multiple casualties
+### Prioritization
+### Assistance
+## Treatments
+### Bandages
+  ACE provides us with four kinds of bandages for different situations. They are:
+
+  - basic bandages (field dressing)
+  - packing bandages 
+  - elastic bandages
+  - quick clots 
+
+  Each bandage has it's own use, but for our case (multiplayer missions where we have medics) we mostly care about 2 of these: packing and elastic bandages. These 2 bandages have very good stats.............
+  TODO: explain shortly what bandage stats actually do
```

Comment:
and also explain why basic bandages & quickclots are very bad and a complete waste of time :D